gandalf.js

```diff
@@ -2,51 +2,48 @@
  * Extras Plugin
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!